Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM node:14-alpine
 WORKDIR /app
 COPY . .
 
-RUN apk add --no-cache --virtual .build-deps alpine-sdk python git openjdk8-jre
+RUN apk add --no-cache --virtual .build-deps alpine-sdk python2 git openjdk8-jre
```

Comment:
Curious what the cause of this break was? An breaking change in the latest alpine image? If so, seems like a good opportunity to pin the docker image major versions to prevent breaking changes like these. E.g. `FROM node:14-alpine` to `FROM node:14-alpine3.13`

Reply:
I'm guessing the `python` package alias was removed from `https://pkgs.alpinelinux.org/`.
Also @rafaelcr, I think the `python` alias pointed to `python3` rather than `python2`. Not sure how much it matter in this project though. It may have only been needed to install `git`, for example.

Reply:
Whoops... thanks @CharlieC3 I'll keep that in mind. When I do the build publish v2 PR I'll revisit these Dockerfiles to see if we can version them better and adjust the python version (or remove it if not needed).